src/test/scripts/README.md

```diff
@@ -1,33 +1,59 @@
+`enterprise/$PR_BRANCH` (even though in git commands you would reference it as
+`origin/$PR_BRANCH`). This branch is currently the same as `enterprise-master`.
+First to make review easier, you should merge community master into it. This
+should apply without any merge conflicts:
 
 ```bash
-git checkout enterprise-master
-git pull # make sure enterprise-master in your local is up-to-date
-git checkout X
-git merge origin/enterprise-master # assuming origin is the remote of citus-enterprise
+git merge community/master
 ```
+Now you need to merge `community/$PR_BRANCH` to `enterprise/$PR_BRANCH`. Solve
+any conflicts and make sure to remove any parts that should not be in enterprise
+even though it doesn't have a conflict, on enterprise repository:
 
-- You should push this to enterprise and open a PR so that the job on community will see this branch. Also this will trigger CI to verify changes are correct.
-- You should get approval for the merge conflict changes on `enterprise/X`, preferably from the same reviewer as they are familiar with the change.
-- You should rerun the `check-merge-to-enterprise` check on `community/X`. You can use re-run from failed option in circle CI.
-- You can now merge `community/X` to `community/master`.
-- You can merge `enterprise/X` to `enterprise-master`.
-- Manually merge `community master` to `enterprise-master`. [https://github.com/citusdata/citus-enterprise/blob/enterprise-master/CONTRIBUTING.md#merging-community-changes-onto-enterprise](https://github.com/citusdata/citus-enterprise/blob/enterprise-master/CONTRIBUTING.md#merging-community-changes-onto-enterprise)
-- You can use `git checkout --ours <conflicting file names>` for all conflicting files. This will use local/our version to resolve the conflicts.
-- Now you can push to `enterprise-master`. Now you can rerun the check on `community/master` and it should pass the check.
+```bash
+git merge "community/$PR_BRANCH"
+```
 
-The subsequent PRs on community will be able to pass the `check-merge-to-enterprise` check as long as they don't have a conflict with `enterprise-master`.
+1. You should push this branch to the enterprise repo. This is so that the job
+   on community will see this branch.
+2. Wait until tests on `enterprise/$PR_BRANCH` pass.
+3. Create a PR on the enterprise repo for your `enterprise/$PR_BRANCH` branch.
+4. You should get approval for the merge conflict changes on
+   `enterprise/$PR_BRANCH`, preferably from the same reviewer as they are
+   familiar with the change.
+5. You should rerun the `check-merge-to-enterprise` check on
+   `community/$PR_BRANCH`. You can use re-run from failed option in circle CI.
+6. You can now merge the PR on community. Be sure to NOT use "squash and merge",
+   but instead use the regular "merge commit" mode.
+7. You can now merge the PR on enterprise. Be sure to NOT use "squash and merge",
+   but instead use the regular "merge commit" mode.
```

Comment:
If we don't merge `community master` to `enterprise-master` wouldn't community master show conflicts? That is why we had the part after https://github.com/citusdata/citus/pull/3918/files#diff-eeb78b71523d6f236eefedabf221e3b9L29

Reply:
By merging the enterprise PR you will have effectively done that already.